Makefile.toml

```diff
@@ -755,6 +755,12 @@ if [[ -n ${containers} ]]; then
   docker stop ${containers}
 fi
 
+# Wait for a while for all components to gracefully exit to dump profiling data for coverage.
+if [[ -n $LLVM_PROFILE_FILE ]]; then
+  echo "Waiting for 5 seconds for coverage data to be dumped..."
```

Comment:
We may install a signal handler to call 

```rust
unsafe extern "C" {
    fn __llvm_profile_write_file() -> libc::c_int;
}
```

to make it more robust in the future.